### ISSUE TITLE
pdf and eps antialiasing / hinting

### ISSUE THREAD

**Issue by klayman** [mate-desktop/atril#173]
When rendering pdf files antialiasing for curves changes depending on the zoom level:
![pdf-aa-1](https://cloud.githubusercontent.com/assets/350162/11777216/6c832726-a27f-11e5-8a58-cfd0cc0fdb71.png)
![pdf-aa-2](https://cloud.githubusercontent.com/assets/350162/11777221/71acc82e-a27f-11e5-91ad-5dd63b0b991b.png)
For small scales antialiasing becomes ragged and edgy, while for large scales it's smooth. The turning-point zoom level differs from file to file - for one pdf-file it is between 50 and 75%, for another - between 150 and 175%.

For eps files curves rendering seems to be of the first (ragged) type regardless of zoom.

Are there any antialiasing settings to tweak?

**Comment by monsta** [comment]
Can you check if it's the same in Evince? It might be something in the libraries/backends.

**Comment by klayman** [comment]
Yes, it is. And seemingly a curve becomes ragged in a pdf when its rendered width is less than 1px - some hinting rules should come into play here. Are there any options in the corresponding libraries to completely disable hinting for curves?

**Comment by monsta** [comment]
I'm not aware of such options, sorry. If Evince exhibits the same behavior, it's better to ask developers of the underlying libraries (probably cairo).
